buckw

```diff
@@ -213,10 +213,15 @@ setupBuckRun ( ) {
 
 # Handle parameters and flags
 handleParams ( ) {
-    # Go directly to kill. Do not run okbuck.
-    if [[ "kill" == $1 ]]; then
-        SKIP_OKBUCK=true
-    fi
+    for arg in "$@"
```

Comment:
It is easier to do

if ($1 == kill) or ( $@ == *"--help"*)

To avoid having to iterate over all the arguments